test/java/org/opendatakit/briefcase/export/Scenario.java

```diff
@@ -79,10 +75,6 @@ static Model createField(DataType dataType) {
     return createField(dataType, "field", null);
   }
 
-  static Model createField(DataType dataType, String fieldName) {
```

Comment:
new/revisit

These test methods could be handy and also serve as an explanation of how to create those things. Also, since it's an internal test helper, not so important to leave it even if it's not used.